### PR TITLE
First version of the paper for tomorrow's bibliography club

### PR DESCRIPTION
**Pull request by maylis-j**
Don't hesitate to note whenever something is unclear :)

### CODE REVIEW

File: collections/_posts/2023-06-30-Shape_Aware_organ_segmentation.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+![](/collections/images/shape_aware_organ_seg/single_org_results.jpg)
+
+*Figure 3: Qualitative segmentation comparison on the hippocampus testing set*
+
+![](/collections/images/shape_aware_organ_seg/single_org_metrics.jpg)
+
+*Table 1: Quantitative comparison of segmentation models on the hippocampus dataset (the methods' names correspond to the loss chosen)*
+
+
+* Compared to using only the segmentation map, the results have less false positives, and reduce the Hausorff Distance (HD)
```
Comment (juliapg) [inline collections/_posts/2023-06-30-Shape_Aware_organ_segmentation.md:111]: Hausdorff

File: collections/_posts/2023-06-30-Shape_Aware_organ_segmentation.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+As the Heaviside function is not differentiable, they use an approximated version during training:
+
+$$f(z)=\frac{1}{1+\exp{(-z\times k)}}$$
+
+where the larger k, the closer the approximation. They use $$k = 1500$$.
+
+During inference, they use the Heaviside not approximated.
+
+**Note**: the authors also tried with two output branches but had better results with only one output
```
Comment (Morgane-des-Ligneris) [inline collections/_posts/2023-06-30-Shape_Aware_organ_segmentation.md:73]:
```suggestion
> **Note**: the authors also tried with two output branches but had better results with only one output
```

File: collections/_posts/2023-06-30-Shape_Aware_organ_segmentation.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+![](/collections/images/shape_aware_organ_seg/multi_org_metrics.jpg)
+
+*Table 2: Dice comparison on the MICCAI 2015 testing set*
+
+![](/collections/images/shape_aware_organ_seg/multi_org_metrics_2.jpg)
+
+*Table 3: comparison of proposed methods on the MICCAI 2015 testing set*
+
+**Note**: Average Symmetric Surface Distance (ASD or ASSD) is the average distance between boundary points from the predicted mask and from the ground truth mask.
```
Comment (Morgane-des-Ligneris) [inline collections/_posts/2023-06-30-Shape_Aware_organ_segmentation.md:130]:
```suggestion
> **Note**: Average Symmetric Surface Distance (ASD or ASSD) is the average distance between boundary points from the predicted mask and from the ground truth mask.
```

**Review by Morgane-des-Ligneris — CHANGES_REQUESTED** [review]
very clear :) !

**Review by Morgane-des-Ligneris — APPROVED** [review]

**Pull request merged**